magento/magento2
# magento/magento2#8171: Price not saved when creating product via REST API

### Description (*)

The change covers adding a default typeId set to **simple** for new products created if no other is sent in post request.

### Related Pull Requests
https://github.com/magento/partners-magento2ee/pull/345
https://github.com/magento/partners-magento2b2b/pull/349


### Fixed Issues (if relevant)

1. magento/magento2#8171: Price not saved when creating product via REST API
2. Fixes https://github.com/magento/magento2/issues/13639
2. Fixes https://github.com/magento/magento2/issues/30487


### Manual testing scenarios (*)

1. Do a POST request to /products with a minimum payload to create a product.
```
{
"product": {
"sku": "test",
"name": "test",
"price": 12.5,
"attribute_set_id": 4
}
}
```
2. Created product should have visible and set price.

### Questions or comments


### Contribution checklist (*)


### Resolved issues:
1. [x] resolves magento/magento2#29504: magento/magento2#8171: Price not saved when creating product via REST API
